google-common/src/test/scala/akka/stream/alpakka/google/PaginatedRequestSpec.scala

```diff
@@ -2,52 +2,44 @@
  * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
  */
 
-package akka.stream.alpakka.googlecloud.bigquery.impl
+package akka.stream.alpakka.google
 
 import akka.actor.ActorSystem
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
 import akka.http.scaladsl.model.HttpMethods.GET
 import akka.http.scaladsl.model.HttpRequest
-import akka.http.scaladsl.model.Uri.Query
-import akka.http.scaladsl.model.headers.OAuth2BearerToken
-import akka.stream.alpakka.googlecloud.bigquery.impl.auth.CredentialsProvider
-import akka.stream.alpakka.googlecloud.bigquery.scaladsl.Paginated
-import akka.stream.alpakka.googlecloud.bigquery.{BigQueryAttributes, BigQuerySettings, HoverflySupport}
+import akka.stream.alpakka.google.scaladsl.Paginated
 import akka.stream.scaladsl.Sink
 import akka.testkit.TestKit
 import io.specto.hoverfly.junit.core.SimulationSource.dsl
 import io.specto.hoverfly.junit.dsl.HoverflyDsl.service
 import io.specto.hoverfly.junit.dsl.ResponseCreators.success
 import org.scalatest.BeforeAndAfterAll
+import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpecLike
 import spray.json.{JsObject, JsString, JsValue}
 
 import scala.concurrent.duration._
-import scala.concurrent.{Await, ExecutionContext, Future}
 
 class PaginatedRequestSpec
     extends TestKit(ActorSystem("PaginatedRequestSpec"))
     with AnyWordSpecLike
     with Matchers
     with BeforeAndAfterAll
+    with ScalaFutures
     with HoverflySupport {
 
   override def afterAll(): Unit = {
     TestKit.shutdownActorSystem(system)
     super.afterAll()
   }
 
-  implicit val settings = BigQuerySettings().copy(credentialsProvider = new CredentialsProvider {
-    override def projectId: String = ???
-    override def getToken()(implicit ec: ExecutionContext, settings: BigQuerySettings): Future[OAuth2BearerToken] =
-      Future.successful(OAuth2BearerToken("yyyy.c.an-access-token"))
-  })
+  implicit val patience = PatienceConfig(timeout = 3.seconds)
```

Comment:
If this is being changed it would be nice to use the Akka test kit `remainingOrDefault` with an `Assert` so the timeout can be scaled with configuration.  Not a big deal since I wouldn't expect this to ever timeout.

https://doc.akka.io/api/akka/2.6/akka/testkit/TestKit.html#remainingOrDefault:scala.concurrent.duration.FiniteDuration

Reply:
Actually I explicitly set this because I did have some timeout issues because Hoverfly was slow to spin up I think. Would `remainingOrDefault` still work for this?

Reply:
`remainingOrDefault` will return a dilated value of the `akka.test.single-expect-default` configuration from the Akka testkit, which is actually 3 seconds by default. Sometimes it's useful to apply a multiple with `akka.test.timefactor` when the test is running in a slow environment (i.e. a build server). It's just a convenient way to increase the timeouts without changing any code.

Reply:
Cool, will be making more use of this!